### DISCUSSION TITLE
Setting up a ZIP drive image, and adding file to it

### DISCUSSION THREAD

**Discussion by arjepsen** [akuker/RASCSI#924] [General]
Hey.
I'm trying to update the firmware on my Fostex VF16.
The way to do it was to attach a SCSI ZIP drive, with the firmware file on the disk.

I think I figured out how to make a ZIP image file, using the web interface to RASCSI, but I can't quite figure out how to mount this image, in order to transfer the firmware file to it.

Any help?

Regards
Anders

**Comment by grichter** [comment]
Eh.. you'll probably have to ssh in so you can mount the image on the pi in order to create an fs and copy files in.

**Reply by rdmark** [reply]
Correct, this is how you do it. Mount the image file at a Linux mount point and transfer the file through the file system. You may have to partition and format the image file with a file system that your host can read, e.g. FAT32.
